live-site/components/welcome-section/team-race/TeamCar.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { TeamCarProps } from '../../../lib/types';
+import { StyledTeamCar, StyledTeamCarContainer } from './Team.styles';
+import car from '../../../../shared-ui/images/van.svg';
```

Comment:
<img width="522" alt="Screenshot 2023-01-18 at 1 25 54 PM" src="https://user-images.githubusercontent.com/59738880/213263772-9a1a195d-1dee-4e13-8436-cf780a4d9fea.png">

In the design, each team has a different colored van so add the other vans in as well